php/Exchange.php

```diff
@@ -1180,6 +1181,8 @@ public function __construct($options = array()) {
         if ($this->markets) {
             $this->set_markets($this->markets);
         }
+
+        $this->after_construct();
```

Comment:
I think for now we can call the method directly instead of this wrapper

Reply:
The thing it that, as I mentioned in toppost, I will be adding some more separate things, and it's best to have that wrapper to avoid a juice mix of codes in one specifically named method.
that's why I made a bit more abstraction, so we have now:
```
afterConstruct () {
    this.method_that_does_X_related_things();
   // we can later add:
   // this.method_that_does_Y_related_things();
}
```